charts/astronomer/templates/commander/commander-deployment.yaml

```diff
@@ -93,3 +93,7 @@ spec:
             - name: COMMANDER_AIRGAPPED
               value: "true"
             {{- end }}
+            {{ if .Values.global.features.namespacePools.enabled }}
```

Comment:
Interesting... `.global.features` could be a great mechanic to abstract things behind, and account for which features we have.

Reply:
Yeah, Constance has good ideas about this !